tests/unit/store/createStore.ts

```diff
@@ -429,12 +429,12 @@ registerSuite({
 
 	'async storage': {
 		'async operation should not be done immediately.'(this: any) {
```

Comment:
Aren't all the operations async? just curious what this test really proves? basically we've set an artificial delay on the async API and then asserted that the delay occurs.